Automattic/jetpack
# VideoPress: Updates to wp-admin media detail view

This PR cleans up a few things in the VideoPress media detail view.

* Removes the `Transcoding Status` meta box, we show conversion progress in the player now so this is no longer needed:
<img width="581" alt="Screen Shot 2021-02-24 at 1 59 52 PM" src="https://user-images.githubusercontent.com/789137/109181945-b781ff00-7749-11eb-923a-e4fefa76b985.png">
* Fixes a bug where the VideoPress player wouldn't display as the video was converting, because the `width` and `height` of the iFrame were being set to `0`:

Old:
<img width="1322" alt="Screen Shot 2021-02-25 at 8 46 08 AM" src="https://user-images.githubusercontent.com/789137/109182116-e7c99d80-7749-11eb-9e75-d56325598f09.png">

New:
<img width="1324" alt="Screen Shot 2021-02-25 at 8 52 10 AM" src="https://user-images.githubusercontent.com/789137/109182140-eef0ab80-7749-11eb-988d-eb92d18e4cf1.png">

* It also fixes a bug where vertical videos would cause scrolling in the detail view, they are now limited to 100% height:

<img width="1293" alt="Screen Shot 2021-02-25 at 9 09 18 AM" src="https://user-images.githubusercontent.com/789137/109182242-062f9900-774a-11eb-81ce-abf156d752a2.png">

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Upload a new video to VideoPress in the wp-admin media library (should be fairly large so conversion takes a while).
* Open the detail view after it uploads (Might take a page refresh, I wonder if we can fix this?) - You should see the player and the conversion message.
* Confirm that you see no `Transcoding Status` metabox. (Sidenote: My dev site never showed the meta box, is there a bug here?)

#### Proposed changelog entry for your changes:
* Bug fixes for managing videos in wp-admin.
